.gitignore

```diff
@@ -58,3 +58,6 @@ Makefile.local
 
 # scan-build artifacts
 scan-build/
+
+# compile_and_test_for_boards default "results" directory
+results/*
```

Comment:
This would make the directory also ignored if it is in sub directories.

```suggestion
results/
```

Reply:
Addressed and directly squashed